crates/matrix-sdk/src/encryption/backups/mod.rs

```diff
@@ -1282,4 +1254,60 @@ mod test {
 
         server.verify().await;
     }
+
+    async fn mock_backup_exists(server: &MockServer) -> MockGuard {
```

Comment:
These should become part of the `MatrixMockServer`.

Reply:
Done in 0d7bbb4472